Scheduler/observing_blocks_maintel/AOS-OR4/BLOCK-276.json

```diff
@@ -0,0 +1,202 @@
+{
+    "name": "BLOCK-276",
+    "program": "BLOCK-276",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "run_command.py",
+            "standard": true,
+            "parameters": {
+                "component": "CCCamera",
+                "cmd": "play",
+                "parameters": {
+                    "playlist": "or4_day1_aos_sweep_cam_dz_v1",
+                    "repeat": true
+                }
+            }
+        },
+        {
+            "name": "maintel/slew_to_azel.py",
+            "standard": true,
+            "parameters": {
+                "track_azel": {
+                    "az": 9.0,
+                    "el": 75.0
+                },
+                "rot_value": 0,
+                "rot_type": "Physical"
+            }
+        },
+        {
+            "name": "maintel/offset_camera_hexapod.py",
+            "standard": true,
+            "parameters": {
+                "z": -150.0
+            }
+        },
+        {
+            "name": "maintel/take_image_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exp_times": 30.0,
+                "note": "Focus Sweep Camera dZ -150um",
+                "filter": "i_01",
+                "image_type": "ACQ",
+                "nimages": 1,
+                "program": "$program"
+            }
+        },
+        {
+            "name": "maintel/offset_camera_hexapod.py",
+            "standard": true,
+            "parameters": {
+                "z": 37.5
```

Comment:
Units again (I'll stop marking these individually here though).

I assume this is the _relative_ shift and not a seek to an absolute encoder value?

Reply:
Yes it's the relative shift